load_debug_symbols/README.md

```diff
@@ -0,0 +1,12 @@
+Load Debug Symbols
+==================
+
+Loads a debug symbol file at the right addresses for .text, .data and .bss sections.
+
+Usage: load-debug-symbols PATHTOFILE
```

Comment:
Use single backtips around commands to make them render correctly.

Reply:
Done.